### DISCUSSION TITLE
Mailersend as Mail alternative

### DISCUSSION THREAD

**Discussion by DevRaeph** [NiclasvanEyk/initializer-for-laravel#224] [Ideas]
Can you please add Mailersend as another point under mail.

There is an official package for the driver on Github.
https://github.com/mailersend/mailersend-laravel-driver

**Comment by NiclasvanEyk** [comment]
I am a bit hesitant to add packages not explicitly mentioned in the Laravel docs. This was the inclusion criteria for the current set of packages and if I start adding new ones by request, I feel like this could get out of hand.

Is there a good reason for including this package (e.g. does it provide something unique, that none of the officially documented mail transports provide)?
